OCP-Infrastructure/.gitignore

```diff
@@ -0,0 +1,20 @@
+/HogaramaInfrastructure/secrets/secrets.yml
```

Comment:
Why are so many special files ignored? If they are generated, shouldn't they be in a work directory?

Reply:
certain yaml files residing in OCP-Infrastructure/[secrets/resources/helm] are templated via jinja2. As such only the template is checked in and the processed yaml is not.

Correct, a working directory that we can exclude as a whole would be beneficial. Thanks for the feedback 👍 

We are in the process of phasing those files out. Currently we keep them as reference to helmify everything based on a working set of resources. Once we have established helm to role out Hogarama, those files will be deleted.